distribution/docs/src/main/resources/content/_managing/_configuring/configuring-multi-factor-auth.adoc

```diff
@@ -0,0 +1,24 @@
+:title: Configuring Multi Factor Authentication
+:type: subConfiguration
+:status: published
+:parent: Configuring REST Services for Users
+:summary: Configuring Multi Factor Authentication using Keycloak as an IdP.
+:order: 23
+
+// This is for MFA
```

Comment:
❓ is this needed

Reply:
Not necessarily, I just copy-pasted a different docs section and they had a similar comment there. It can easily be deleted.